README.md

```diff
@@ -103,8 +103,9 @@ OBject
 1. Fork it (http://github.com/yuki24/did_you_mean/fork)
 2. Create your feature branch (`git checkout -b my-new-feature`)
 3. Commit your changes (`git commit -am 'Add some feature'`)
-4. Push to the branch (`git push origin my-new-feature`)
-5. Create new Pull Request
+4. Confirm all tests passed (`bundle exec rake`)
```

Comment:
`Make sure all tests pass` sounds more natural to me. Could you change this line?